modules/editor/inserter/button.js

```diff
@@ -0,0 +1,14 @@
+import Inserter from './';
+
+const h = wp.element.createElement;
+
+const InserterButton = ( { opened, onClick } ) => {
+	return h( 'div', { className: 'inserter__button' },
+		h( 'a', { onClick },
+			h( 'span', { className: 'dashicons dashicons-plus' } )
```

Comment:
Do you think we ought to create a component for Dashicons, assuming this will be a very common pattern? Or is it obvious enough to apply the classes? Where would said component live if we were to introduce it?